### PR TITLE
update README

### PR DESCRIPTION
**Pull request by mango-gremlin**
## Motivation
Readme should reflect current state of the game.

## What was changed/added
So far, I only changed the cover pic, removed the "contributing" part (can put it back in if you want) and added links to our releases and an itch link I will fill in when the game is uploaded there.

## Additional Information
**If you have any suggestions on what to add to the readme, please say so. I have offered to edit it, but I dont really know what else to add.**

Should I add:
- ~small game tutorial using the tutorial slides from aylin? to explain what you're doing?~
- write a little bit abt our process/include the stuff that will be in our presentation?
- structure of what can be found where (eg wiki etc)
- game trailer link?
- screenshots
- that this was originally a game jam & what the theme was
- feedback from playtesters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +32,9 @@ We are using an MIT License for this game. For more information, see [here](http
 - Raffael Rack
 - Katja Bochniak
 - Mara Cockburn
-- to be expanded later
+
+### Sound Effects and Music
```
Comment (ScatteredDrifter) [inline README.md:36]: reasonable

File: README.md [hunk 2]
```diff
@@ -2,32 +2,24 @@
...
 ## How to run the game
-~ watch this space ~
-
----
-
-## Contributing
-
-### How to submit issues
--> guide on how to make a bug report? -> maybe irrelevant but still nice to have in there, add this in time
+Find our most recent release [here](https://github.com/mango-gremlin/arch-enemies/releases) - it should contain an .exe file you can download and play on your computer.
```
Comment (ScatteredDrifter) [inline README.md:21]: yup!

File: README.md [hunk 3]
```diff
@@ -40,7 +32,9 @@ We are using an MIT License for this game. For more information, see [here](http
 - Raffael Rack
 - Katja Bochniak
 - Mara Cockburn
-- to be expanded later
+
+### Sound Effects and Music
+- Adam König
 
 ### Font
 yoster island - by [codeman38](http://www.zone38.net/ )
```
Comment (ScatteredDrifter) [inline README.md:40]: great :)

**Review by ScatteredDrifter — APPROVED** [review]
looks good to me :) 
I would not add the tutorial --> this belongs in the game oor Wiki I would argue?

**Review by ScatteredDrifter — APPROVED** [review]
looks good to me, declutters the main-page and structures it in short sections. 
The ToC provided is nice to have too :)

**Comment by ScatteredDrifter** [comment]
Considering your list of things to add: 
```
Should I add:

    small game tutorial using the tutorial slides from aylin? to explain what you're doing?
    write a little bit abt our process/include the stuff that will be in our presentation?
    structure of what can be found where (eg wiki etc)
    game trailer link?
    screenshots
    that this was originally a game jam & what the theme was
    feedback from playtesters

```
I would argue that the current version contains the important parts we ought to cover while the rest: 
- does not belong into this README -> but wiki or similar 
- is not necessary / covered elsewhere already

File: README.md [hunk 4]
```diff
@@ -1,30 +1,78 @@
...
+If you don't own a windows device or do not want to download questionable executables from github, **you can find the game on [itch](https://mango-gremlin.itch.io/arch-enemies)**.
+
+If you run into any problems while playing, see [here](#reporting-a-problem-or-bug)
 
 ---
 
-## Licensing
-We are using an MIT License for this game. For more information, see [here](https://github.com/mango-gremlin/arch-enemies/blob/d390d485051dfb2a5ab55d9eb6f97ea2c08d81c3/LICENSE).
+## Reporting a Problem or Bug
+If you run into any problems while playing, do not hesitate to tell us!
```
[the remark this answers is not in the document]
Comment (mango-gremlin) [reply README.md:77]: you're welcome! i just linked the discussion i automatically created with the release, as everyone can post there and i think thats a good place to gather. other people have also already posted feedback on itch actually!! very exciting!!!

**Comment by mango-gremlin** [comment]
> Considering your list of things to add:
> 
> ```
> Should I add:
> 
>     small game tutorial using the tutorial slides from aylin? to explain what you're doing?
>     write a little bit abt our process/include the stuff that will be in our presentation?
>     structure of what can be found where (eg wiki etc)
>     game trailer link?
>     screenshots
>     that this was originally a game jam & what the theme was
>     feedback from playtesters
> ```
> 
> I would argue that the current version contains the important parts we ought to cover while the rest:
> 
>     * does not belong into this README -> but wiki or similar
> 
>     * is not necessary / covered elsewhere already

very fair, i also admit i did not update this very well ah
but yes i also think everything else is not necessary, the current state is good (except for the video links which will be added later)

**Pull request merged**